docs/runtime_attributes.md

```diff
@@ -0,0 +1,22 @@
+# Key Runtime Attributes
+
+This doc is intended to capture the key runtime attributes for Node.js.  
+These attributes will be used along with the use-cases in order to
+guide what we develop/run as part of the community benchmarking
+efforts.
+
+# Attributes
+
+The following attributes have been identified so far:
+
+* startup/stop time
+* memmory footprint at startup
+* memory footprint after load
```

Comment:
maybe something like, after a few cycles of the event loop? if the user is reading in files or something async then it won't show up until just after the load metrics are collected. not sure how to properly measure this, but possibly trigger on idle or similar.